### ISSUE TITLE
New action on user level to set email as verified

### ISSUE THREAD

**Issue by amergey** [auth0-extensions/auth0-delegated-administration-extension#189]
The issue #36 
_The Auth0 dashboard allows you to mark a user's email address as being verified. This is an action we also need in the dashboard._
has been closed as won't fix, why ?  I think it is something that could be useful

**Comment by fxnicolas** [comment]
Same here, with the use case: Our support team has to _manually mark as verified_ users when the verification emails are blocked by their server. And of course, we cannot grant our support team admin.

**Comment by amergey** [comment]
Yes this is still relevant. I find a bit annoying to set it as wontfix without any explanations.
Maybe there are good reasons not to fix this, but as it was asked by several people, it could be good to know them

**Comment by raymondware** [comment]
V3 of this extension will allow you to do this by using a combination of the settings query hook and write hook.

In your settings query hook field, you would do something similar to this:
```
function(ctx, callback) {
    return callback(null, {
       userFields: [
           {
            label: "Email Verified",
            property: "email_verified",
            display: true,
            edit: {
              type: "checkbox"
            },
           }
       ]
    });
}
```
Then in the write hook you would do something like this:
```
function(ctx, callback) {
  var data = {
    ...ctx.payload,
    email_verified: ctx.payload.email_verified
  };

  return callback(null, data);
}
```

**Comment by amergey** [comment]
Yes still relevant this, as the action is provided in auth0 dashboard, it is expected to have same capabilities from this extension

**Comment by amergey** [comment]
It is still relevant

**Comment by amergey** [comment]
It is still relevant

**Comment by mdahlman** [comment]
This is a problem for me. I need to force validation for some users that have problems receiving some emails.
I have access to only the user dashboard, so I'm stuck.

**Comment by amergey** [comment]
still relevant
